File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/NotificationsResponse.java
package dev.ragnarok.fenrir.api.model.response;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import dev.ragnarok.fenrir.api.model.VKApiCommunity;
import dev.ragnarok.fenrir.api.model.VKApiUser;
import dev.ragnarok.fenrir.api.model.feedback.VKApiBaseFeedback;

public class NotificationsResponse {
    @SerializedName("count")
    public int count;

    @Nullable
    @SerializedName("items")
    public List<VKApiBaseFeedback> notifications;

    @Nullable
    @SerializedName("profiles")
    public List<VKApiUser> profiles;

    @Nullable
    @SerializedName("groups")
    public List<VKApiCommunity> groups;

    @Nullable
    @SerializedName("next_from")
    public String nextFrom;

    @SerializedName("last_viewed")
    public long lastViewed;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/NotificationsResponse.kt
package dev.ragnarok.fenrir.api.model.response

import dev.ragnarok.fenrir.api.model.VKApiCommunity
import dev.ragnarok.fenrir.api.model.VKApiUser
import dev.ragnarok.fenrir.api.model.feedback.VKApiBaseFeedback
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class NotificationsResponse {
    @SerialName("count")
    var count = 0

    @SerialName("items")
    var notifications: List<VKApiBaseFeedback>? = null

    @SerialName("profiles")
    var profiles: List<VKApiUser>? = null

    @SerialName("groups")
    var groups: List<VKApiCommunity>? = null

    @SerialName("next_from")
    var nextFrom: String? = null

    @SerialName("last_viewed")
    var lastViewed: Long = 0
}